api/envoy/service/tap/v2alpha/common.proto

```diff
@@ -24,6 +26,7 @@ message TapConfig {
   // a tap will occur and the data will be written to the configured output.
   OutputConfig output_config = 2 [(validate.rules).message.required = true];
 
+  envoy.api.v2.core.RuntimeFractionalPercent filter_enabled = 3;
```

Comment:
Docs? What is this doing specifically? Name?

Reply:
similar to other filters, it allows to gradually enabled the filter